src/untangled/client/impl/application.cljc

```diff
@@ -218,20 +218,22 @@
                                       (swap! initial-state assoc :ui/locale "en-US")
                                       initial-state)
                                     (assoc initial-state :ui/locale "en-US"))
-        config                    {:state       initial-state-with-locale
-                                   :send        (fn [tx cb]
-                                                  (server-send (assoc app :reconciler @rec-atom) tx cb))
-                                   :migrate     (or migrate tempid-migrate)
-                                   :normalize   true
-                                   :remotes     remotes
-                                   :pathopt     pathopt
-                                   :merge-ident (fn [reconciler app-state ident props]
-                                                  (update-in app-state ident (comp sweep-one merge) props))
-                                   :merge-tree  (fn [target source]
-                                                  (merge-handler mutation-merge target source))
-                                   :parser      parser
-                                   :shared      shared}
-        rec                       (om/reconciler config)]
+        reconciler-options        (update reconciler-options :migrate #(or % tempid-migrate))
+        config                    (merge {:migrate     nil
```

Comment:
Could we just delete line 221, put tempid-migrate in the first merge map, and delete the :shared nil entry? That would make the data flow a bit more clear, and would eliminate noise.